hrpc/list.go

```diff
@@ -0,0 +1,50 @@
+package hrpc
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/tsuna/gohbase/pb"
+)
+
+// GetTableNames models a GetTableNames pb call
+type GetTableNames struct {
+	base
+	regex            string
+	includeSysTables bool
+	namespace        string
+}
+
+// NewGetTableNames creates a new GetTableNames request that will list
+// tables in hbase
+func NewGetTableNames(ctx context.Context, table string, regex string) *GetTableNames {
```

Comment:
do you mean "namespace" instead of "table"? Otherwise I don't see where you pass namespace, maybe add a test for that.

Reply:
These were a TODO in the previous commit. Modified them to use the option pattern and added tests to ensure that works in the integration test.